.travis.yml

```diff
@@ -45,6 +46,7 @@ matrix:
     env: USE_CMAKE=YES
 
 script:
+  - if [ ${TRAVIS_OS_NAME} = 'linux' ]; then KWStyle -R -v -gcc -d src; fi
```

Comment:
What are the problems currently in the tests directory? Can you run the tests directory with the current configuration so that I may see the issues in Travis-CI? From there it may be better to determine if they should be fixed or are OK as is.